parsers/ELEXON.py

```diff
@@ -11,13 +11,12 @@
 """
 
 import re
-from datetime import date, datetime, time, timedelta
+from datetime import date, datetime, time, timedelta, timezone
 from io import StringIO
```

Comment:
More a curiosity/learning for me than a change request: Why are we using `timezone` here? Shouldn't it make sense to import ZoneInfo as we do for the other zones and get `TIMEZONE = ZoneInfo("Europe/London")` from there?

Reply:
timezone from datetime have a singleton with a static definition for utc, this is the recommended approach by the python documentation.